sycl/source/detail/queue_impl.hpp

```diff
@@ -422,15 +417,12 @@ class queue_impl {
   const async_handler MAsyncHandler;
   const property_list MPropList;
 
-  RT::PiQueue MCommandQueue = nullptr;
-
   /// List of queues created for FPGA device from a single SYCL queue.
   vector_class<RT::PiQueue> MQueues;
   /// Iterator through MQueues.
-  size_t MQueueNumber = 0;
+  size_t MNextQueueID = 0;
```

Comment:
Is it index or really some ID? If it's still index it should be named as index or iterator.

Reply:
I think it's index, will rename.